packages/client/src/raffles/__fixtures__/postRaffleParticipation.texture.ts

```diff
@@ -0,0 +1,18 @@
+import { raffleId } from 'tests/__fixtures__/raffles';
+import { rest, RestHandler } from 'msw';
+import type { RaffleParticipation } from '../types';
+
+const path = `/api/checkout/v1/raffles/${raffleId}/participations`;
```

Comment:
Should this be
```suggestion
const path = '/api/checkout/v1/raffles/:raffleId/participations';
```

Is there a reason for a different approach?

Reply:
no, there isn't..
pls double check